### PR TITLE
docs(hooks): Overriding Form Scripts

### PR DESCRIPTION
**Pull request by abhishekbalam**
Added docs for overriding Form Scripts

image

### CODE REVIEW

File: frappe_docs/www/docs/user/en/python-api/hooks.md [hunk 1]
```diff
@@ -797,6 +797,33 @@ class CustomToDo(ToDo):
...
+		frm.trigger('my_custom_code');
+	},
+	my_custom_code: function(frm){
+		console.log(frm.doc.name)
+	}
+});
+```
+
+> The functions defined in the mentioned file in `doctype_js` dictionary 
+> will override those in the respective standard form script.
```
Comment (netchampfaris) [inline frappe_docs/www/docs/user/en/python-api/hooks.md:825]: Can we make this more clear?

File: frappe_docs/www/docs/user/en/python-api/hooks.md [hunk 2]
```diff
@@ -797,6 +797,33 @@ class CustomToDo(ToDo):
 > It is recommended that you extend the standard class of the doctype, otherwise
 > you will have to implement all of the core functionality.
 
+### Override Form Scripts
```
Comment (netchampfaris) [inline frappe_docs/www/docs/user/en/python-api/hooks.md:800]: Also, this needs to be added to the table at the bottom of the page

**Review by netchampfaris — CHANGES_REQUESTED** [review]

File: frappe_docs/www/docs/user/en/python-api/hooks.md (hunk 2 above)
Comment (abhishekbalam) [reply frappe_docs/www/docs/user/en/python-api/hooks.md:800]: Done

File: frappe_docs/www/docs/user/en/python-api/hooks.md (hunk 1 above)
Comment (abhishekbalam) [reply frappe_docs/www/docs/user/en/python-api/hooks.md:825]: Is this better?

image

**Review by netchampfaris — APPROVED** [review]

**Pull request merged**